src/amo/constants.js

```diff
@@ -42,3 +42,4 @@ export const FEATURED_ADDONS_TO_LOAD = 25;
 // Number of add-ons in the featured, trending, and highest rated landing page
 // sections.
 export const LANDING_PAGE_ADDON_COUNT = 4;
+export const LANDING_PAGE_THEME_ADDON_COUNT = 3;
```

Comment:
Should this just be `LANDING_PAGE_THEME_COUNT`? The `ADDON` part seems redundant. Perhaps the one above should also be changed to `LANDING_PAGE_EXTENSION_COUNT`?